src/TuringTutorials.jl

```diff
@@ -1,186 +1,134 @@
 module TuringTutorials
 
-import IOCapture
+import Requires
+import Weave
 
-using IJulia
-using InteractiveUtils
-using Pkg
-using Plots
-using Requires
-using Weave
+import InteractiveUtils
+import Markdown
+import Pkg
 
-const REPO_DIR = string(pkgdir(TuringTutorials))::String
+const REPO_DIR = dirname(@__DIR__)
+const CSS_FILE = joinpath(REPO_DIR, "templates", "skeleton_css.css")
+const LATEX_FILE = joinpath(REPO_DIR, "templates", "julia_tex.tpl")
 
-include("cache.jl")
-include("build.jl")
-
-export build_folder, tutorial_path, folder2filename
-export build, build_and_exit, verify_logs, tutorials, changed_tutorials
-
-# Not building PDF, because it is fragile. Maybe later.
-default_build_list = (:script, :html, :github, :notebook)
-
-cssfile = joinpath(@__DIR__, "..", "templates", "skeleton_css.css")
-latexfile = joinpath(@__DIR__, "..", "templates", "julia_tex.tpl")
+const DEFAULT_BUILD = (:script, :html, :github)
 
 function __init__()
-    @require Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80" include("weaveplots.jl")
-end
-
-"""
-    tutorial_path(folder)
-
-Return the absolute path to a tutorial `folder`.
-"""
-tutorial_path(folder) = joinpath(REPO_DIR, "tutorials", folder)
-
-function polish_latex(path::String)
-    # TODO: Is it maybe better to overload https://github.com/JunoLab/Weave.jl/blob/b5ba227e757520f389a6d6e0f2cacb731eab8b12/src/WeaveMarkdown/markdown.jl#L10-L17
-    # and replace the `tex.formula` there? Only negative part is that this of course
-    # will affect all of the markdown parsers, which is not necessarily desirable.
-    txt = open(f -> read(f, String), path)
-    open(path, "w+") do f
-        txt = replace(txt, "\\\\\n" => "\\\\\\\\\n")
-        write(f, txt)
-    end
+    Requires.@require Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80" include("weaveplots.jl")
```

Comment:
```suggestion
    @require Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80" include("weaveplots.jl")
```
Together with previous suggestion.